### DISCUSSION TITLE
Filter by short/feature

### DISCUSSION THREAD

**Discussion by peachscissors** [GoodbyteCo/Letterboxd-Watchlist-Picker#78] [Ideas]
Hello! I love this project. If it wouldn't be too difficult to implement it would be cool to be able to filter by either short, feature or both. Other filters available on Letterboxd (decade, genre, etc.) could be cool too, but length I feel would be the most useful :)

**Comment by HoloPollock** [comment]
Hey thanks for the recommendation we will look in to it but I have a feeling it may be harder than expected just based on how we get the film data.

**Comment by HoloPollock** [comment]
Just to update the feature is being worked on bigger update than initially thought so just need to fix small small bugs introduced and then add the UI but its coming *Probably*

**Reply by peachscissors** [reply]
Wow y'all work quickly! Thanks for taking my suggestion :) 

P.S. Love your profile pic

**Reply by HoloPollock** [reply]
Sorry for the delay it got a little more complicated that expected but 🤞 should be out in the next few days

**Reply by HoloPollock** [reply]
Its up and running now hope you like it

**Reply by peachscissors** [reply]
Excellent, thank you!

**Comment by ghost** [comment]
There should be a filter for decade and genre as well

**Reply by HoloPollock** [reply]
sorry we are still scarping letterboxd so this feature is pretty challenging to do. Get them to give me api acess and Ill do it :p
